pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -231,23 +231,32 @@ func (k *Bootstrapper) init(cfg config.ClusterConfig) error {
 		return errors.Wrap(err, "run")
 	}
 
-	if cfg.Driver == driver.Docker {
-		if err := k.applyKicOverlay(cfg); err != nil {
-			return errors.Wrap(err, "apply kic overlay")
+	go func() {
+		if cfg.Driver == driver.Docker {
+			if err := k.applyKicOverlay(cfg); err != nil {
+				glog.Warningf("unable to apply kic overlay network: %v", err)
+			}
 		}
-	}
+	}()
 
-	if err := k.applyNodeLabels(cfg); err != nil {
-		glog.Warningf("unable to apply node labels: %v", err)
-	}
+	go func() {
```

Comment:
I'd add a waitgroup here to make sure these functions finish and avoid possible race conditions

[example here](https://github.com/kubernetes/minikube/blob/9c1dbe745c5e8441a2ce3bddeffd64ebe1c564f6/pkg/minikube/command/ssh_runner.go#L78-L94)

Reply:
thank you, done !